### ISSUE TITLE
Suspend release? (due to mismatch of guids in translated decks)

### ISSUE THREAD

**Issue by aplaice** [axelboc/anki-ultimate-geography#394]
In light of the `guid`s [not matching in the translated decks](https://github.com/axelboc/anki-ultimate-geography/issues/391#issuecomment-777310776):

> The issue is caused by the fact that the `guid`s have all changed in all the translated versions of the decks (which we had not realised...).
>
> I believe that this is because of the difference in how `anki-dm` and `brain-brew` work with `guid`s.
>
> `anki-dm` would take the `guid` from `src/data.csv`, apply some transformation to it (don't remember the details) and use that in the generated `deck.json`.
>
> `brain-brew` just takes the `guid` in `src/data/main.csv` without any modification.
>
> Hence, when we switched from `anki-dm` to `brain-brew` we replaced the `guid`s in `src/data/main.csv` with the generated values for the English deck.
>
> What I didn't realise until now is that `anki-dm`'s `guid` transformation was **language-specific**. (This was presumably so that people could use multiple translated decks in parallel, which is pretty convenient!)
>
> (In my partial defence, the transformation is the same for the "normal" and "extended" decks and I didn't realise that this was different for the translated decks. (Logically, though, there's no reason to use both the "normal" and "extended" decks in parallel, while there is reason to use translated decks in parallel.)
>
> This means that in the translated decks, none of the new `guid`s match the old ones...

and since our intention is for minor upgrades to not lose progress, should we add a warning message to the 4.1 release, at the top, to the effect of, say:

```markdown
### **Please do not use any of the translated decks in this release until we investigate further.**

Please [see here](https://github.com/axelboc/anki-ultimate-geography/issues/394) for details.
```

or, perhaps:

```markdown
### **Please do not use the Spanish, German, French or Norwegian decks if you're upgrading, until we investigate further.**

Please [see here](https://github.com/axelboc/anki-ultimate-geography/issues/394) for details.
```

(though I'm not sure whether, in the long run, we do want the ability of using translated decks in parallel — in which case we wouldn't people to use **any** of the translated decks).

(Also, I'm not sure if having people using the ES,DE,FR and NB decks be on a mix of the old and new GUIDs would be useful, so perhaps just drop the "if you're upgrading" part? OTOH we don't really want to stop new people from using these translations...)

<hr/>

Apart from trying to minimise further "damage" (complications), in the short run, there's also obviously the question of how we'll deal with the problem, going forward, though I'm not really sure whether that discussion should be here or in a separate issue.

**Comment by axelboc** [comment]
Done!

**Comment by axelboc** [comment]
I've removed the ZIPs of the translated decks form the release.

**Comment by ohare93** [comment]
> 
> 
> I've removed the ZIPs of the translated decks form the release.

You can do that? Nice
